extension/src/popup/helpers/blockaid.ts

```diff
@@ -17,6 +17,11 @@ interface BlockAidScanSiteResult {
   // ...
 }
 
+interface BlockAidScanTxResult {
```

Comment:
Not using the result yet, but when we add the UI pieces of this we can expand this type.